pinot-common/src/main/java/com/linkedin/pinot/common/data/MetricFieldSpec.java

```diff
@@ -23,6 +23,10 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public final class MetricFieldSpec extends FieldSpec {
 
+  // TODO: can this field being serialized?
```

Comment:
Yes, this field can be serialized and deserialized if you define setter and getter correctly.